python/feathub/online_stores/tests/test_mysql_online_store.py

```diff
@@ -0,0 +1,188 @@
+            host=self.mysql_container.get_container_host_ip(),
+            port=int(
+                self.mysql_container.get_exposed_port(
+                    self.mysql_container.port_to_expose
+                )
+            ),
+            database=self.mysql_container.MYSQL_DATABASE,
+            username=self.mysql_container.MYSQL_USER,
+            password=self.mysql_container.MYSQL_PASSWORD,
+            table=self.table_name,
+        )
+
+        engine = sqlalchemy.create_engine(self.mysql_container.get_connection_url())
+        with engine.begin() as conn:
+            conn.execute(
+                sqlalchemy.text(
+                    f"CREATE TABLE {self.table_name} ("
+                    f"id BIGINT PRIMARY KEY, val BIGINT, ts TEXT)"
+                )
+            )
+
+        self.client.materialize_features(file_source, sink, allow_overwrite=True).wait(
+            30_000
+        )
+
+        self.on_demand_feature_view = OnDemandFeatureView(
+            name="on_demand_feature_view",
+            features=[
+                "feature_table.val",
+            ],
+            keep_source_fields=True,
+            request_schema=Schema.new_builder().column("id", types.Int64).build(),
+        )
+
+    def tearDown(self) -> None:
+        super().tearDown()
+        engine = sqlalchemy.create_engine(self.mysql_container.get_connection_url())
+        with engine.begin() as conn:
+            conn.execute(sqlalchemy.text(f"DROP TABLE IF EXISTS {self.table_name}"))
+
+    def test_get_online_features(self):
+        source = MySQLSource(
+            name="feature_table",
+            schema=self.feature_table_schema,
+            host=self.mysql_container.get_container_host_ip(),
+            port=int(
+                self.mysql_container.get_exposed_port(
+                    self.mysql_container.port_to_expose
+                )
+            ),
+            database=self.mysql_container.MYSQL_DATABASE,
+            username=self.mysql_container.MYSQL_USER,
+            password=self.mysql_container.MYSQL_PASSWORD,
+            keys=["id"],
+            timestamp_field="ts",
+            table=self.table_name,
+        )
+
+        self.client.build_features([source, self.on_demand_feature_view])
+
+        request_df = pd.DataFrame(np.array([[2], [3]]), columns=["id"])
+        online_features = (
+            self.client.get_online_features(
+                request_df=request_df,
+                feature_view=self.on_demand_feature_view,
+            )
+            .sort_values(by=["id"])
+            .reset_index(drop=True)
+        )
+
+        expected_result = (
+            pd.DataFrame([[2, 2], [3, 3]], columns=["id", "val"])
+            .sort_values(by=["id"])
+            .reset_index(drop=True)
+        )
+
+        assert_frame_equal(expected_result, online_features)
+
+    def test_more_input_column_than_keys(self):
+        source = MySQLSource(
+            name="feature_table",
```

Comment:
It is inconsistent that we use the string `feature_table` and the variable `self.table_name` interchangeably in this file.